themes/hugo-steam-theme/layouts/partials/head.html

```diff
@@ -19,6 +19,17 @@
       <link href="{{ .RSSLink }}" rel="feed" type="application/rss+xml" title="{{ .Site.Title }}" />
     {{ end }}
 
+    <meta property="og:type" content="article" />
+    <meta property="og:title" content="{{ .Title }}"/>
+    <meta property="og:description" content="{{ .Description | markdownify }}" />
```

Comment:
Could you add a default in case people is sharing a listing? Otherwise it will be rendered as empty. I think you can use the one used in `<meta description>` [Line#12](https://github.com/src-d/blog/pull/184/files#diff-b546ffe97424802fe480d896f4300f65R12)
```html
{{ .Site.Params.description }}
```

Reply:
fixed